framework/src/problems/ReferenceResidualProblem.C

```diff
@@ -81,9 +81,10 @@ ReferenceResidualProblem::ReferenceResidualProblem(const InputParameters & param
   else if (params.isParamValid("reference_vector"))
     _reference_vector = &_nl->getVector(getVectorTagID(getParam<TagName>("reference_vector")));
   else
-    mooseError("Either the `reference_residual_variables` or `reference_vector` parameter must be "
-               "specified for `ReferenceResidualProblem`. `reference_residual_variables` is "
-               "deprecated so we recommend using `reference_vector`");
+    mooseInfo("Neither the `reference_residual_variables` nor `reference_vector` parameter is "
```

Comment:
So the idea is that reference residual problem will be the same as nominal behavior if not variables are set, rather than error out?

Reply:
If you look at the overall PR, I didn't make any changes here. `reference_residual_variables` has never been a required parameter. There are existing augmented lagrange contact tests that do not set this parameter.

I don't know what you mean by nominal behavior. Do you mean like `FEProblem` convergence criteria? If that's what you mean, then no it is not the same. If the parameter is not set, then like the mooseInfo says you'll be relying on the absolute tolerance. No relative tolerance checks will be performed. So ya I can't error out here without breaking existing tests; moreover, I don't even want to make this a warning because downstream apps may be relying on this (rather bad) behavior and if they have `allow_warnings` in their test root, then I will break their testing.

Reply:
Hm, maybe there have been some changes to this I don't get until I get hands on.